app/views/keywords/show.html.slim

```diff
@@ -0,0 +1,47 @@
+div class="d-flex flex-column align-items-center mt-4"
+
+  h1.display-3 == keyword.keyword
+
+  - if keyword.status == 'completed'
+    div class="d-flex mt-5 w-100 justify-content-center"
+      div id="topPosAdwordsCount" class="card"
+        div class="card-body" 
+          .display-4 = keyword.top_pos_adwords
+        div class="card-info" = t('keyword.top_pos_adwords').upcase
```

Comment:
another thing, in slim, we no need to define the `div`

instead of `div class="card-info" = t('keyword.top_pos_adwords').upcase`

we can do 

`.card-info = t('keyword.top_pos_adwords').upcase`

Reply:
Just for my understanding. Did you mean we can do `.card-info = t('keyword.top_pos_adwords').upcase` when the div has only 1 class?

Reply:
Fixed in 42644abbb9a15b2d0eebf420eb65d700cf5f9a5d